### PR TITLE
[INLONG-4197][Manager] Add Hbase sink info and load node utils

### LINKED ISSUE #4197: [Feature][Manager] Add Hbase sink info and load node utils
### Description

Adaptive Hbase load node 

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by LvJiancheng**
### Title Name: [INLONG-4197][Manager] Adaptive Hbase load node 

where *XYZ* should be replaced by the actual issue number.

Fixes #4197 

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve?*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/sink/HbaseSink.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import org.apache.inlong.manager.common.enums.DataFormat;
+import org.apache.inlong.manager.common.enums.SinkType;
+import org.apache.inlong.manager.common.pojo.stream.SinkField;
+import org.apache.inlong.manager.common.pojo.stream.StreamSink;
+
+import java.util.List;
+
```
Comment (woofyzhao) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/sink/HbaseSink.java:32]: Please add some java doc before classes and public/static methods.

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/sink/HbaseSink.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    @ApiModelProperty("Sink buffer flush max rows")
+    private String sinkBufferFlushMaxRows;
+
+    @ApiModelProperty("Sink buffer flush interval")
+    private String sinkBufferFlushInterval;
+
+    @ApiModelProperty("Create table or not")
+    private boolean needCreated;
+
+    @ApiModelProperty("Field definitions for hive")
```
Comment (gong) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/sink/HbaseSink.java:70]: `hive` should be `hbase`. Please attention annotate.

**Review by gong — COMMENTED** [review]
Add some UT for new data source

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/sink/HbaseSink.java (hunk 2 above)
Comment (LvJiancheng) [reply inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/sink/HbaseSink.java:70]: done

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/sink/HbaseSink.java (hunk 1 above)
Comment (LvJiancheng) [reply inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/sink/HbaseSink.java:32]: ok

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/hbase/HbaseSinkOperation.java [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+            fieldEntity.setInlongGroupId(groupId);
+            fieldEntity.setInlongStreamId(streamId);
+            fieldEntity.setSinkType(sinkType);
+            fieldEntity.setSinkId(sinkId);
+            fieldEntity.setIsDeleted(GlobalConstants.UN_DELETED);
+            entityList.add(fieldEntity);
+        }
+
+        sinkFieldMapper.insertAll(entityList);
+        LOGGER.info("success to save hive field");
```
Comment (woofyzhao) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/hbase/HbaseSinkOperation.java:133]: not hive

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/sink/HbaseSink.java [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+import java.util.List;
+
+/**
+ * Hbase sink.
+ */
+@Data
+@EqualsAndHashCode(callSuper = true)
+@NoArgsConstructor
+@AllArgsConstructor
+@ApiModel("Hive sink configuration")
```
Comment (woofyzhao) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/sink/HbaseSink.java:40]: not Hive

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/hbase/HbaseSinkOperation.java (hunk 3 above)
Comment (LvJiancheng) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/hbase/HbaseSinkOperation.java:133]: done

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/sink/HbaseSink.java (hunk 4 above)
Comment (LvJiancheng) [reply inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/sink/HbaseSink.java:40]: done

**Review by woofyzhao — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/util/InlongStreamSinkTransfer.java [hunk 5]
```diff
@@ -207,6 +212,65 @@ private static StreamSink parseKafkaSink(KafkaSinkResponse sinkResponse, StreamS
         return kafkaSink;
     }
 
+    private static SinkRequest createHbaseRequest(StreamSink streamSink, InlongStreamInfo streamInfo) {
```
Comment (healchow) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/util/InlongStreamSinkTransfer.java:215]: Please add Java doc for the static method, thanks.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/util/LoadNodeUtils.java [hunk 6]
```diff
@@ -161,6 +165,44 @@ public static HiveLoadNode createLoadNode(HiveSinkResponse hiveSinkResponse) {
         );
     }
 
+    public static HbaseLoadNode createLoadNode(HbaseSinkResponse hbaseSinkResponse) {
```
[the remark this answers is not in the document]
Comment (LvJiancheng) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/util/LoadNodeUtils.java:173]: done

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/util/InlongStreamSinkTransfer.java (hunk 5 above)
Comment (LvJiancheng) [reply inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/util/InlongStreamSinkTransfer.java:258]: done

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/util/InlongStreamSinkTransfer.java [hunk 7]
```diff
@@ -207,6 +212,65 @@ private static StreamSink parseKafkaSink(KafkaSinkResponse sinkResponse, StreamS
...
+        hbaseSinkRequest.setProperties(hbaseSink.getProperties());
+        if (CollectionUtils.isNotEmpty(hbaseSink.getSinkFields())) {
+            List<SinkFieldRequest> fieldRequests = createSinkFieldRequests(hbaseSink.getSinkFields());
+            hbaseSinkRequest.setFieldList(fieldRequests);
+        }
+        return hbaseSinkRequest;
+
+    }
+
+    private static StreamSink parseHbaseSink(HbaseSinkResponse sinkResponse, StreamSink sink) {
```
[the remark this answers is not in the document]
Comment (LvJiancheng) [reply inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/util/InlongStreamSinkTransfer.java:290]: done

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/hbase/HbaseSinkOperation.java [hunk 8]
```diff
@@ -0,0 +1,233 @@
...
+        entity.setStatus(SinkStatus.CONFIG_ING.getCode());
+        entity.setModifier(operator);
+        entity.setModifyTime(new Date());
+        sinkMapper.updateByPrimaryKeySelective(entity);
+
+        boolean onlyAdd = SinkStatus.CONFIG_SUCCESSFUL.getCode().equals(entity.getPreviousStatus());
+        this.updateFieldOpt(onlyAdd, hbaseSinkRequest);
+
+        LOGGER.info("success to update sink of type={}", sinkType);
+
```
[the remark this answers is not in the document]
Comment (LvJiancheng) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/hbase/HbaseSinkOperation.java]: done

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/sink/HbaseSink.java [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+    private String sinkBufferFlushInterval;
+
+    @ApiModelProperty("Create table or not")
+    private boolean needCreated;
+
+    @ApiModelProperty("Field definitions for hbase")
+    private List<SinkField> sinkFields;
+
+    @ApiModelProperty("Data format type for stream sink")
+    private DataFormat dataFormat;
```
Comment (healchow) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/sink/HbaseSink.java:77]: Did this filed needed for HBase sink?

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sink/hbase/HbaseSinkDTO.java [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+@AllArgsConstructor
+public class HbaseSinkDTO {
+
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+
+    @ApiModelProperty("Target table name")
+    private String tableName;
+
+    @ApiModelProperty("Namespace")
+    private String nameSpace;
```
Comment (healchow) [inline inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sink/hbase/HbaseSinkDTO.java:48]: Please change to `namespace`, thanks.

**Review by healchow — CHANGES_REQUESTED** [review]

**Review by healchow — APPROVED** [review]

**Pull request merged**